tests/terraform_compliance/steps/its_key_value_test_cases/v_case.json

```diff
@@ -0,0 +1,22 @@
+{
+    "stash": [
+        {
+            "id": "a",
+            "foo": "BaR"
+        },
+        {
+            "id": "b",
+            "foo": "BaR[0]"
+        },
+        {
+            "id": "c",
+            "foo": ["BaR"]
```

Comment:
We're case sensitive inside string lists.

Reply:
These comments are very useful, would be nice to integrate these comments within the tests.

I thought we were testing case sensitive matching here 😊 We need to support case insensitive matching everywhere, maybe except parameter keys defined in values

Reply:
> maybe except parameter keys defined in values

I'm not sure what you mean by this.

Reply:
I've fixed the case issue.